### ISSUE TITLE
502 Bad Gateway

### ISSUE THREAD

**Issue by alphv** [modoboa/modoboa-installer#208]
# Impacted versions

* Distribution: Debian 9.4
* Codename: Stretch (stable)
* Arch: 64 Bits

# Problem
I'm getting 502 bad Gateway 

What I did : giving "777" perm to /srv/modoboa still not work.
Doing "pip install --no-binary modoboa modoboa" still not work
Have the latest python version.
Error I get : 
```
2018/04/28 16:01:41 [error] 9078#9078: *34 upstream prematurely closed connection while reading response header from upstream, client: 86.XXX.XXX.XXX, server: mail.xoxo.moe, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:/run/uwsgi/app/modoboa_instance/socket:", host: "mail.xoxo.moe"

2018/04/28 16:01:41 [error] 9078#9078: *34 upstream prematurely closed connection while reading response header from upstream, client: 86.XXX.XXX.XXX, server: mail.xoxo.moe, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:/run/uwsgi/app/modoboa_instance/socket:", host: "mail.xoxo.moe", referrer: "https://mail.xoxo.moe/"

2018/04/28 16:01:47 [error] 17425#17425: *40 upstream prematurely closed connection while reading response header from upstream, client: 86.XXX.XXX.XXX, server: mail.xoxo.moe, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:/run/uwsgi/app/modoboa_instance/socket:", host: "mail.xoxo.moe"

2018/04/28 16:01:48 [error] 17425#17425: *40 upstream prematurely closed connection while reading response header from upstream, client: 86.XXX.XXX.XXX, server: mail.xoxo.moe, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:/run/uwsgi/app/modoboa_instance/socket:", host: "mail.xoxo.moe", referrer: "https://mail.xoxo.moe/"
```

**Comment by alphv** [comment]
Tried to reinstall it but had again some another errors : 

`2018/04/28 16:55:58 [error] 23882#23882: *2 upstream prematurely closed connection while reading response header from upstream, client: 86.225.104.109, server: mail.xoxo.moe, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:/run/uwsgi/app/modoboa_instance/socket:", host: "mail.xoxo.moe"
2018/04/28 16:55:59 [error] 23882#23882: *2 upstream prematurely closed connection while reading response header from upstream, client: 86.225.104.109, server: mail.xoxo.moe, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:/run/uwsgi/app/modoboa_instance/socket:", host: "mail.xoxo.moe", referrer: "https://mail.xoxo.moe/"
`

**Comment by altergui** [comment]
As far as i understand, this will be solved when 1.10.5 is released,
until then (i bumped into the same problem as you), i'm doing this workaround https://github.com/modoboa/modoboa-installer/issues/197#issuecomment-385621392 to have a usable install
hope that helps in the meantime :)
